certified-connectors/DocuSign/script.csx

```diff
@@ -1471,13 +1464,46 @@ public class Script : ScriptBase
   private string GetEnvelopeUrl(JToken envelope)
   {
     var uriBuilder = new UriBuilder(this.Context.Request.RequestUri);
-    var envelopeUrl = uriBuilder.Uri.ToString().Contains("demo") ?
-      "https://apps-d.docusign.com/send/documents/details/" + envelope["envelopeId"] :
-      "https://app.docusign.com/documents/details/" + envelope["envelopeId"];
+    var envelopePath = uriBuilder.Uri.ToString().Contains("demo") ?
+        "/send/documents/details/" + envelope["envelopeId"]
+      : uriBuilder.Uri.ToString().Contains(".mil") ?
+        "/documents/details/" + envelope["envelopeId"]
+      : uriBuilder.Uri.ToString().Contains("stage") ?
+        "/send/documents/details/" + envelope["envelopeId"]
+      : "/documents/details/" + envelope["envelopeId"];
+    var envelopeUrl = GetApplicationURL() + envelopePath;
 
     return envelopeUrl;
   }
 
+  private string GetApplicationURL()
```

Comment:
consider renaming to `GetDocusignApiBaseUri`

Reply:
Done